File: languages/python/flipper/led.py
import lf

funcs = dict(
	_led_configure = 1,
	_led_rgb = 0,
)

def rgb(r, g, b):
	lf.invoke("led", funcs['_led_rgb'], lf.types['void'], [lf.uint8(r), lf.uint8(g), lf.uint8(b)])

File: library/swift/Flipper/LED.swift
public struct LED {
    let module: Module

    public init(flipper: Flipper) {
        self.module = Module(name: "led", device: flipper)
    }

    public func rgb(_ r: UInt8, _ g: UInt8, _ b: UInt8) throws {
        try module.invoke(index: 0, args: [r, g, b])
    }

    public func configure() throws {
        try module.invoke(index: 1, args: [])
    }
}

extension Flipper {
    public var led: LED {
        return LED(flipper: self)
    }
}
